conman/routes/handlers.py

```diff
@@ -53,6 +75,26 @@ class RouteViewHandler(BaseHandler):
 
     Views will receive `route` as a keyword arg.
     """
+    @classmethod
+    def check(cls, route):
+        """Ensure route has a sensible view attribute."""
+        route_name = route.__name__
+        if not hasattr(route, 'view'):
+            return [checks.Error(
+                '{} must have a `view` attribute.'.format(route_name),
+                hint='This is a requirement of {}.'.format(cls.__name__),
+                obj=route,
+            )]
+
+        if not isinstance(vars(route)['view'], staticmethod):
```

Comment:
Unfortunately `route.view != vars(route)['view']`.

I could add a comment, but, given I'm right on the edge of my understanding here, I'm not sure what it should say! Suffice to say this is the only way I have found to determine that an attribute of a class (rather than an instance) is defined at a `staticmethod`.